pmm-tests/ps-specific-tests-2-x.bats

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bats
+
+## mysql
+
+
+MYSQL_USER='root'
+
+@test "run pmm-admin under regular(non-root) user privileges" {
+if [[ $(id -u) -eq 0 ]] ; then
+	skip "Skipping this test, because you are running under root"
+fi
+run pmm-admin
+echo "$output"
+    [ "$status" -eq 1 ]
+    [ "${lines[0]}" = "usage: pmm-admin [<flags>] <command> [<args> ...]" ]
+}
+
+
+@test "run pmm-admin under root privileges" {
+if [[ $(id -u) -ne 0 ]] ; then
+	skip "Skipping this test, because you are NOT running under root"
+fi
+run pmm-admin
+echo "$output"
+    [ "$status" -eq 1 ]
+    [ "${lines[0]}" = "usage: pmm-admin [<flags>] <command> [<args> ...]" ]
+}
+
+
+@test "run pmm-admin add mysql based on running intsances" {
+	  COUNTER=0
+		for i in $(pmm-admin list | grep "MySQL" | sed 's|.*(||;s|)||') ; do
+      let COUNTER=COUNTER+1
+			run pmm-admin add mysql --username=${MYSQL_USER} localhost:3306 mysql_$COUNTER

```

Comment:
@puneet0191  would it be better to use some $PORT variable , so we can use this test for MySQL instance is installed on specified port.